internal/cmd/test-bridged-tap/main.go

```diff
@@ -38,8 +38,12 @@ import (
 )
 
 func main() {
-	skel.PluginMain(add, check, del,
-		// support CNI versions that support plugin chaining
+	skel.PluginMainFuncs(
```

Comment:
I don't see any changes to the cni package version, why'd we change from `PluginMain` to `PluginMainFuncs`?

Reply:
PluginMain is marked deprecated (re: https://pkg.go.dev/github.com/containernetworking/cni/pkg/skel#PluginMain) and the linter update required for toolchain update is starting to enforce no deprecated function calls.